torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1198,6 +1198,23 @@ def sample_inputs_linalg_matrix_power(op_info, device, dtype, requires_grad):
 
     return inputs
 
+def sample_inputs_logical_binaryops(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
This is interesting. What's your thinking on creating a new function to provide sample inputs for these operations vs reusing `sample_inputs_binary_pwise`? 

https://github.com/pytorch/pytorch/blob/b606df2beea388190dbaeb8d51271bdc30d13a7d/torch/testing/_internal/common_methods_invocations.py#L1652

Reply:
Thanks @mruberry, that's a good catch! I was not aware of `sample_inputs_binary_pwise` function.  I will be using this in my next commit. Looks like it covers all the test cases that are used in `logical_or`, `logicaal_and`, `logical_xor`.